clients/client/src/test/java/org/projectnessie/client/http/TestHttpClient.java

```diff
@@ -100,16 +196,14 @@ void testReadTimeout() {
   @Test
   void testPut() throws Exception {
     ExampleBean inputBean = new ExampleBean("x", 1, NOW);
-    HttpHandler handler =
-        h -> {
-          Assertions.assertEquals("PUT", h.getRequestMethod());
-          assertThat(h.getRequestHeaders())
-              .containsEntry("Content-Encoding", Collections.singletonList("gzip"));
-          try (InputStream in = new GZIPInputStream(h.getRequestBody())) {
+    TestServer.RequestHandler handler =
+        (req, resp) -> {
+          Assertions.assertEquals("PUT", req.getMethod());
+          try (InputStream in = req.getInputStream()) {
             Object bean = MAPPER.readerFor(ExampleBean.class).readValue(in);
             Assertions.assertEquals(inputBean, bean);
           }
-          h.sendResponseHeaders(200, 0);
+          emptyResponse(resp);
```

Comment:
imo this should be named `writeEmptyResponse`, as is this statement look odd
also for consistency with `writeResponseBody`